resonate/bridge.py

```diff
@@ -341,7 +345,7 @@ def _process_delayed_events(self) -> None:
                 for item in events:
                     match item:
                         case Function(id, cid, func):
-                            self._cq.put_nowait(Return(id, cid, self._handle_function(func)))
+                            self._processor.enqueue(id, cid, func)
```

Comment:
I think a better pattern for the processor would be to take a `func` and a `callback` that way the processor does not need to take on a dependency for the bridge cq and does not need to know about the id and cid the func comes from. Something like:
```suggestion
    def cb(result):
        self.cq.put_nowait(Return(id, cid, result))
     
    self._processor.enqueue(func, cb)
```

Reply:
Yeah. But the the bridge would needs to pull the cqe from the processor to run the callback. Would you prefer that?

I'm fine either way. I do understand passing the dependency couple things much more, but it's simpler IMO